modules/bamtools/stats/main.nf

```diff
@@ -0,0 +1,35 @@
+process BAMTOOLS_STATS {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
This should be process_single now we have that.

Reply:
"should"? I kind of missed that. nf-core/tools version 2.5.1 suggests "process_low" as default

Reply:
Sorry, I should have been more explicit. The `process_single` tag is a new one that has been recently added, so that isn't present in tools (or the documentation by the looks of it). It only requests one core by default, instead of the two that `process_low` requests.

Reply:
To be precise, these are the default options for the process tag:
https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config#L29

Reply:
If the tool is going to work fine with 6GB of RAM, then single is the better choice for a program that doesn't use more than one core.

Reply:
super @SPPearce thanks! the process is single threaded, but wasn't sure about the memory. I've checked the profiles of execution on my cluster (exomes of different species) and it should be fine. just committed the change.